content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Console Improvements, February 2020"
+h1: "Pulumi Console Improvements, February 2020"
+authors: ["chris-smith"]
+tags: ["pulumi","new-features"]
+date: "2020-02-28"
+meta_desc: "Recent improvements to the Pulumi Console: stack tags, audit logs, CI/CD integration, downloadable checkpoint files"
```

Comment:
Since we will have future posts like this, should we put February 2020 in the meta description? So that users searching for things like "February 2020 pulumi changelog" or what ever and this comes up. Just a thought..not sure if it actually matters to have the description the way I am thinking.

Reply:
The meta description doesn't have a direct effect on rank, but rather is used to help describe to the user what the page is about. It would have an impact on CTR (click-through-rate) but are not a factor in the ranking algorithm.

https://moz.com/learn/seo/meta-description

Reply:
Doesn't the meta_description also show-up in the search result?

Reply:
Yes, it shows up in the search result below the page title. So the result today would look something like:

[Pulumi Console Improvements, February 20202 | Pulumi](#)
Recent improvements to the Pulumi Console: stack tags, audit logs, CI/CD integration, downloadable checkpoint files.

Reply:
Got it. Yes, moreover, the page title itself has "February 2020" already, so what I was thinking originally doesn't matter anymore.